tensorboard/webapp/runs/store/runs_reducers.ts

```diff
@@ -36,16 +36,17 @@ import {
   RunsUiRoutefulState,
   RunsUiState,
 } from './runs_types';
-import {serializeExperimentIds} from './utils';
+import {groupRuns, serializeExperimentIds} from './utils';
 
 const {
   initialState: dataInitialState,
   reducers: dataRouteContextReducers,
 } = createRouteContextedState<RunsDataRoutefulState, RunsDataRoutelessState>(
   {
-    colorOverride: new Map(),
-    defaultColor: new Map(),
-    nextGroupColorIndex: 0,
+    runColorOverride: new Map(),
+    defaultRunColor: new Map(),
+    groupToColor: new Map(),
```

Comment:
It's a bit hard to tell these two maps from the naming.
IIUC, `defaultRunColor` means the mapping of run to color
while `groupToColor` is the mapping of group to color, a group could be a run or a set of runs (like group by experiment)
Can we rename this to be more a pair-like?

Reply:
Renamed the property name to be more explicit and left a comment in the interface. PTAL.